en/gps_compass/rtk_gps_locosys_r2.md

```diff
@@ -0,0 +1,109 @@
+# LOCOSYS Hawk R2 RTK-Compass GPS
+
+The [LOCOSYS Hawk R2](https://www.locosystech.com/en/product/hawk-r2.html) is a dual-frequency [RTK GPS module](../gps_compass/rtk_gps.md) receiver designed for compatibility with Pixhawk.
+Additionally, it is equipped with an e-compass.
+The only difference between this module and the [LOCOSYS Hawk R1](rtk_gps_locosys_r1.md) is the presence of an E-compass on Hawk R2.
+
+The module can act as rover when installed on the aircraft.
+The receiver is capable of concurrently tracking all global civil navigation systems, including GPS, GLONASS, GALILEO, BEIDOU and QZSS.
+It acquires both L1 and L5 signals at the same time while providing the centimeter-level RTK positioning accuracy.
+
+The built-in lightweight helical antenna enhances RTK positioning stability.
+The fast time-to-first-fix, RTK convergence, superior sensitivity, low power consumption make it a better choice for Pixhawk-based platform UAVs.
+
+Main features include:
+- Centimeter-level RTK high position accuracy+ integrated 3-axis e-compass
+- Concurrent reception of L1 and L5 band signals
+- Support GPS, GLONASS, BEIDOU, GALILEO, QZSS
+- Capable of SBAS (WAAS, EGNOS, MSAS, GAGAN)
+- Support 135-channel GNSS
+- Fast TTFF at low signal level
+- Free hybrid ephemeris prediction to achieve faster cold start
+- Default 5Hz, up to 10 Hz update rate (SBAS support 5Hz only)
+- Build-in super capacitor to reserve system data for rapid satellite acquisition
+- Build-in 3 axis compass function
+- Three LED indicator for Power, PPS and Data transmit
+
+:::note
+This module can be used with PX4 v1.13 or above.
+:::
+
+![LOCOSYS Hawk R2](../../assets/hardware/gps/locosys_hawk_a1/locosys_hawk_a1_gps.png)
+
+
+## Where to Buy
+
+* [LOCOSYS Hawk R2](https://www.locosystech.com/en/product/hawk-r2.html)
+
+## Kit Contents
+
+An RTK GPS kit includes:
+- 1x GPS Module
+- 1x Helix antenna
+- 1x 6-pin JST-GH 
+
+
+## Configuration
+
+RTK setup and use on PX4 via *QGroundControl* is largely plug and play (see [RTK GPS](../advanced_features/rtk-gps.md) for more information).
+Connect your Hawk R2 to the `GPS2` port on compatible Pixhawk boards (preferred, though you can use any other unused UART port).
+
+For the aircraft, you should set the parameter [SER_GPS2_BAUD](../advanced_config/parameter_reference.md#SER_GPS1_BAUD) to 230400 8N1 to ensure that PX4 configures the correct baudrate.
+
+## Wiring and Connections
+
+Hawk R2 RTK GPS comes with an 6 pin JST-GH connector that can be plugged into a Pixhawk autopilot.

```

Comment:
Is this correct?

Reply:
it is normally same as R1, so yes, on Pixhawk side can be connected to 6-pin connectors as Telem and GPS2!